nexus/posts/models.py

```diff
@@ -0,0 +1,39 @@
+# Third Party Stuff
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+# Nexus Stuff
+from nexus.base.models import TimeStampedUUIDModel
+
+CONTENT_TYPE_CHOICES = (('text', 'Text'),
```

Comment:
Are these really needed?

Reply:
We need them for the choices in the fields below. Is there any other alternative of making it look cleaner?

Reply:
I've removed these globals and added the tuples directly to attributes.